tests/test_main_loop.py

```diff
@@ -0,0 +1,15 @@
+import shutil
+import os
+from jupyter_book_to_htmlbook.jupter_book_to_htmlbook import main
```

Comment:
There is a typo in the import here that was throwing an error when I ran the tests. Just need to add a "y" in the second jupyter.

Reply:
Ach -- thanks for the catch. I squished that change into the commit (and thanks for the tip!).